releasenotes/notes/aml-add-file-limit-info-6ee9898dade59359.yaml

```diff
@@ -0,0 +1,20 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    In the agent.log, a DEBUG, WARN, and ERROR log are being added to report
+    how many file handles the core agent process has open. The DEBUG log will
+    report the info, the WARN log will appear when the Core Agent is over 90%
+    of the OS file limit, and the ERROR log will appear when the Core Agent
+    has reached 100% of the OS file limit. In the status.log, fields 
+    CoreAgentProcessOpenFiles and OSFileLimit are being added to the Logs
+    Agent section. 
+issues:
+  - |
+    This feature is currently only being implemented for Linux-based hosts.
```

Comment:
I don't think this is necessary to call out as an "issue". Maybe just a sentence at the end of the feature announcement - "Currently linux-only"